docs/reference/aggregations/bucket/terms-aggregation.asciidoc

```diff
@@ -208,8 +208,13 @@ determined and is given a value of -1 to indicate this.
 [[search-aggregations-bucket-terms-aggregation-order]]
 ==== Order
 
-By default, the `terms` aggregation orders terms by descending document `_count`.
-Use the `order` parameter to specify a different sort order.
+By default, the `terms` aggregation orders terms by descending document
+`_count`.  This produces bounded error, which elasticsearch can report (see
+<<terms-agg-doc-count-error>>.)
+
+You can use the `order` parameter to specify a different sort order.  It is extremely
+easy to create a terms ordering that will just return wrong results, and not
+obvious to see when you have done so.  Change this only with caution.
```

Comment:
Do we want to discourage use of the `order` parameter altogether? Given that it's very easy to create a terms ordering that returns inaccurate results, I wonder if we want to expand on the **WARNING** message on line 219 and just say "don't use this". 

> WARNING: Avoid using the `order` parameter to specify a different sort order. Due to the way
> the `terms` aggregation
> <<search-aggregations-bucket-terms-aggregation-shard-size,gets terms from
> shards>>, sorting by ascending doc count often produces inaccurate results. If you need to
> find rare terms, use the <<search-aggregations-bucket-rare-terms-aggregation,`rare_terms`>>
> aggregation instead.

Reply:
Thanks for the context @not-napoleon. I think your instincts are correct here. We might want to add a little more context about why it's so easy to create a terms ordering that won't return correct results. What can we replace as the [reason] in the following sentence? Is this easily explained?

> You can use the `order` parameter to specify a different sort order, but we don't recommend it. Because [reason], it's easy to create a terms ordering that returns incorrect results. Use this parameter with caution.